engine/src/main/java/org/terasology/rendering/logic/LightComponent.java

```diff
@@ -43,14 +44,29 @@
     @Replicate
     public float lightAmbientIntensity = 1.0f;
 
+    /**
+     * This helps control how focused the specular light is. A smaller number will make a wider cone of light. A larger number will make a narrower cone of light.
+     */
     @Replicate
-    public float lightSpecularPower = 1.0f;
```

Comment:
Just makes me think @Josharias is working on turning the torches into flashlights, bringing us from the medieval age into modern times :-)